enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -79,6 +78,23 @@ def price_for_content(self, content_data, course_run_data):
             )
             content_price = DEFAULT_CONTENT_PRICE
 
+        return content_price
+
+    def price_for_content(self, content_data, course_run_data):
+        """
+        Helper to return the "official" price for content.
+        The endpoint at ``self.content_metadata_url`` will always return price fields
+        as USD (dollars), possibly as a string or a float.  This method converts
+        those values to USD cents as an integer.
+        """
+        content_price = None
+
+        if (content_key := course_run_data.get('key')) and content_data['normalized_metadata_by_run'][content_key]:
```

Comment:
might want to change this around for safety a little, something like:
```python
course_run_key = course_run_data.get('key')
if course_run_key in content_data.get('normalized_metadata_by_run', {}):
    if normalized_price := content_data['normalized_metadata_by_run'][course_run_key].get('content_price'):
        content_price = normalized_price
```

Reply:
Updated 👍🏽